src/includes/enriching-events/user-feedback/example-widget/ruby.rails.mdx

```diff
@@ -0,0 +1,25 @@
+In Rails, being able to serve dynamic (e.g. `.html.erb`) pages in response to errors is required in order to pass the needed `event_id` to the JavaScript SDK. Normally, this looks like overwriting the `config.exceptions_app` setting to point at your own router, and then calling your own controller to handle the error. Several tutorials or gems exist to allow for this.
+
+Once you are able to serve dynamic exception pages, supporting user feedback is straightforward.
```

Comment:
We try to avoid telling people what their experience of carrying out a task will be in case it doesn't end up that way for them :-)

```suggestion
Once you're able to serve dynamic exception pages, you can support user feedback.
```